web/themes/custom/bglobal/source/_patterns/01-molecules/hero/hero.twig

```diff
@@ -0,0 +1,23 @@
+<section class="hero__container">
+  <div class="hero__image__container">
+    {{ hero.img }}
+  </div>
+  <div class="hero__text__container container">
+    <h1 class="hero__text--title h2">
```

Comment:
I would change these elements to `hero__subtitle`, `hero__title`, etc. I don't see a good reason to put `text` in there, unless you had a use for it.